drivers/dai/intel/ssp/dai-params-intel-ipc4.h

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __DAI_PARAMS_INTEL_IPC4_H__
+#define __DAI_PARAMS_INTEL_IPC4_H__
+
+#include <stdint.h>
+
+#define I2S_TDM_INVALID_SLOT_MAP1 0xF
+#define I2S_TDM_MAX_CHANNEL_COUNT 8
+#define I2S_TDM_MAX_SLOT_MAP_COUNT 8
+
+/**< Type of the gateway. */
+enum ipc4_connector_node_id_type {
```

Comment:
This needs to be prefixed to align with the owner/usage. e.g.
```dai_intel_ipc4_...``` for all enums, structs here. This ways it's obvious its for Intel IPC4 only users

Reply:
ok